### DISCUSSION TITLE
Why does react-northstar's background colors differ from the UI guidelines?

### DISCUSSION THREAD

**Discussion by cantdoart5** [microsoft/fluentui#16576] [Q&A]
I'm looking at the native Teams tabs and the UI guidelines.  For the light theme they have a gray background with a white background for the dropdowns and input fields.  I could be using it wrong, but the react-northstar@0.51.4 theme has a whitebackground with gray for the background of the input fields.

Normally, this wouldn't be that big of a deal, but for the dark theme the background for the Teams sidebar is #201F1F, but the provider background is #2D2C2C and the input background is #201F1F.  As a result, I have to add padding to the provider and can't use the existing padding offered in the Sidebar, which gives me even less width to work with (280px - 16px padding) vs just the 280px.

Am I using the theme wrong or should it different from the existing Teams apps and the UI guidelines?

**Comment by rolandbodenstaff** [comment]
I'm wondering the same thing! If this is by design how do you change the background color I can't seems to find it under the theming documentation.
